### ISSUE TITLE
Create separate invitations page for account

### ISSUE THREAD

**Issue by VladislavSokov** [widefix/pocketmoney#9]
Add possibility to manage invitations:

-   create
-   copy link
-   delete in any status (the invitee loses access in this case)

**Comment by ka8725** [comment]
@andreybakanovsky you can start working on this by creating a controller/views and adding tests (they will fail but for awhile until the model added)

**Comment by andreybakanovsky** [comment]
@ka8725 I agree with your statement. The title of the task does not match the content, description. The latter requires decomposition
